<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTabForImage</class>
 <widget class="QWidget" name="CustomTabForImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="originalGrahpic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>386</width>
     <height>288</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>386</width>
     <height>288</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>386</width>
     <height>288</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Znaczniki położenia</string>
   </property>
   <widget class="QCheckBox" name="groundTruthCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rzeczywiste położenie</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="calculatedCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wyliczone położenie</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="face_element">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OI - Elementy twarzy - Wizualizacja</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
